.vscode/extensions.json

```diff
@@ -4,6 +4,7 @@
     "davidanson.vscode-markdownlint",
     "kaellarkin.hugo-shortcode-syntax",
     "budparr.language-hugo-vscode",
-    "bradlc.vscode-tailwindcss"
+    "bradlc.vscode-tailwindcss",
+    "vasubasraj.flashpost"
```

Comment:
kinda a weird extension add. has only 9k installs. but i dont really care much

Reply:
The reason we have flashpost is so that editors using VSCode can more easily retrieve user ids of GitHub users. That's the main reason, I'll probably end up removing it later on if we find another solution for that. Maybe a python script or something.

Reply:
if its just making http requests cant you make an npm script that runs curl